lib/rules/value-keyword-case/__tests__/index.js

```diff
@@ -58,6 +58,32 @@ testRule(rule, {
     code: "a::before { content: \"TeSt tEsT\"}",
     description: "ignore value within quotes",
   }, {
+    code: "a { content: url(http://www.example.com/test.png) ; }",
+  }, {
+    code: "a { content: attr(value-string); }",
+  }, {
+    code: "a { content: attr(vAlUe-StRiNg); }",
+  }, {
+    code: "a { content: attr(VALUE-STRING); }",
+  }, {
+    code: "a { content: counter(counter-name); }",
+  }, {
+    code: "a { content: counter(cOuNtEr-NaMe); }",
```

Comment:
If you've already tested the line about `toLowerCase` (with the `value-string` tests above), I don't think we need to multiply every test with alternate cases.